src/main/scala/edu/cmu/cs/obsidian/parser/Parser.scala

```diff
@@ -265,7 +263,7 @@ object Parser extends Parsers {
         }
 
         val parseTryCatch = TryT() ~! LBraceT() ~! parseBody ~! RBraceT() ~!
-                            CatchT() ~! LBraceT() ~! parseBody <~ RBraceT() ^^ {
+            CatchT() ~! LBraceT() ~! parseBody <~ RBraceT() ^^ {
```

Comment:
This whitespace change isn't great.

Reply:
This is the same story as the other place you pointed out.